### DISCUSSION TITLE
move-focus to accept name of pane

### DISCUSSION THREAD

**Discussion by danielb2** [zellij-org/zellij#2984] [Ideas]
right now the action move-focus command is fairly limited. if there are nested panes it can be difficult to navigate focus to a specific pane, and even with a relatively simple four pane layout, getting from lower right quadrant to top left would require two commands on CLI to do it programatically. 

I propose new panes gets an index just like tabs do that can be used to refer to them so every pane has a number. this already aligns with `go-to-tab`

perhaps a `go-to-pane` would also make sense, but I think move-focus is already sufficient

`action move-focus <direction|index|name>` may be workable

**Comment by cristiand391** [comment]
GH issue: https://github.com/zellij-org/zellij/issues/2779
